### ISSUE TITLE
Accessibility: Multiple elements in the DOM with the same id

### ISSUE THREAD

**Issue by KubaGorka** [highcharts/highcharts#23930]
When accessibility module is imported, styledMode is turned on, and there is more than one chart on the page, there are elements on the page with duplicate id's, violating accessibility rules (E631). 

#### Expected behaviour
Each id on the page should be unique

#### Actual behaviour
Multiple elements with the same id

#### Live demo with steps to reproduce
https://codesandbox.io/p/sandbox/fervent-grass-mhwdl2

Demo is also using "@highcharts/react" npm package, but I believe this issue comes from the Highcharts core.

#### Product version
Highcharts Core 12.4.0

#### Affected browser(s)
All

**Comment by magdalena-gut** [comment]
Hi @KubaGorka , thanks for reporting this. Unfortunately, we weren’t able to reproduce the issue on our side - neither in your demo nor in a simplified vanilla JS example: https://jsfiddle.net/v2Lumkdb/

Are you using any accessibility (a11y) auditing tools that reported this problem? If so, could you let us know which one?

Could you also please double-check whether the issue is reproducible on your side when running the page without any browser extensions, plugins, or similar tools enabled?

**Comment by KubaGorka** [comment]
Hi @magdalena-gut,

Sorry for taking some time to reply. You are right—I missed one important detail. I’ve dug a bit deeper into the issue, and it seems the actual reason is the Highcharts 3D module.

I’ve updated the CodeSandbox, and the issue is now reproducible. I also checked, and importing the 3D script into the simplified example you provided causes the issue to appear as well.

All you need to do is add:
`<script src="https://code.highcharts.com/highcharts-3d.js"></script>`

After that, each chart will have the following element rendered inside it:
`<filter id="highcharts-darker">...</filter>`

**Comment by magdalena-gut** [comment]
Thanks for the clarification - I can now reproduce the issue on my end.

I’m marking this as a bug and passing it on for prioritization. Thanks again for your help in tracking this down!
